src/common/models/Projection.ts

```diff
@@ -309,6 +311,25 @@ export class Projection {
     return false;
   }
 
+  get twoWeekPercentChangeInCases() {
+    return this.getTwoWeekPercentChange(this.smoothedDailyCases);
+  }
+
+  get twoWeekPercentChangeInDeaths() {
+    return this.getTwoWeekPercentChange(this.smoothedDailyDeaths);
+  }
+
+  private getTwoWeekPercentChange(series: any[]): number | null {
+    const lastIndex = indexOfLastValue(series) ?? series.length;
```

Comment:
_nit_: shouldn't it be `series.length - 1`? I see that we compensate below, feel free to  leave as it for now

```ts
const lastIndex = indexOfLastValue(series) ?? series.length - 1;
```

Reply:
Er, yeah, I think I threw that in to make TypeScript happy since `indexOfLastValue()` can return null, but really the series should never be empty and if it is, this code is going to behave strangely so I just added an assert instead.

The `lastIndex + 1` below does need to stay as-is since .slice() treats the end index as exclusive, not inclusive.